src/main/java/com/stripe/net/StripeResponseStream.java

```diff
@@ -22,7 +22,7 @@ public StripeResponseStream(int code, HttpHeaders headers, InputStream body) {
    *
    * @return the StripeResponse
    */
-  StripeResponse unstream() throws IOException {
+  public StripeResponse unstream() throws IOException {
```

Comment:
why?

Reply:
Because we need to change the visibility to use it here in rawRequest:
https://github.com/stripe/stripe-java/pull/1553/files#diff-8e4bf2c082d9f6d0e50bd100e2c612728dbcf75d230b1f4d33bd5adfcec4ef8aR265